src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -109,6 +112,9 @@ def __init__(self, url, max_schemas_per_subject=1000, ca_location=None, cert_loc
         self.url = utils.urldefragauth(self.url)
 
         self._session = s
+        key_password = conf.pop('ssl.key.password', None)
+        self.is_key_password_provided = not key_password
```

Comment:
Add '_'. Change `self.is_key_password_provided` to `self._is_key_password_provided` as this is going to be a private variable.